### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by demis-palma**
Can't return after throw an exception.
In addition, method declared void can't return boolean false.

To be merged on review.

### CODE REVIEW

File: administrator/components/com_associations/views/association/view.html.php [hunk 1]
```diff
@@ -62,15 +62,14 @@ class AssociationsViewAssociation extends JViewLegacy
 	 * @return  void
 	 *
 	 * @since   3.7.0
+	 * @throws Exception
```
Comment (C-Lodder) [inline administrator/components/com_associations/views/association/view.html.php]: Think needs 1 more space

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 023d9caae4bacc18fa66e3fd21b3ec98b0903837<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15424">issues.joomla.org/tracker/joomla-cms/15424</a>.</sub>

**Comment by csthomas** [comment]
I have tested this item :white_check_mark: successfully on 023d9caae4bacc18fa66e3fd21b3ec98b0903837<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15424">issues.joomla.org/tracker/joomla-cms/15424</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**